pkg/tests/ossm/smoke_test.go

```diff
@@ -79,3 +173,39 @@ func assertSidecarInjectedInAllBookinfoPods(t TestHelper, ns string) {
 			}
 		})
 }
+
+func assertSMCPDeploysAndIsReady(t test.TestHelper, ver version.Version) {
+	t.LogStep("Install SMCP")
+	InstallSMCPVersion(t, meshNamespace, ver)
+	oc.WaitSMCPReady(t, meshNamespace, smcpName)
+	oc.ApplyString(t, meshNamespace, GetSMMRTemplate())
+	t.LogStep("Check SMCP is Ready")
+	oc.WaitSMCPReady(t, meshNamespace, smcpName)
+}
+
+// func assertUninstallDeletesAllResources(t test.TestHelper, ver version.Version) {
+// 	t.LogStep("Delete SMCP in namespace " + meshNamespace)
+// 	oc.DeleteFromString(t, meshNamespace, GetSMMRTemplate())
+// 	DeleteSMCPVersion(t, meshNamespace, ver)
+// 	retry.UntilSuccess(t, func(t TestHelper) {
+// 		oc.GetAllResources(t,
+// 			meshNamespace,
+// 			assert.OutputContains("No resources found in",
+// 				"All resources deleted from namespace",
+// 				"Still waiting for resources to be deleted from namespace"))
+// 	})
+// }
```

Comment:
Why not to remove this function? Will you remember about it?
```suggestion
```

Reply:
Hi @jewertow I did not remove because is used in the sub test case that I disable because of this bug: https://issues.redhat.com/browse/OSSM-4064